Core_OverlayMods/Clothes/KoiClothesOverlayGui.cs

```diff
@@ -195,10 +222,14 @@ private void RegisterCustomControls(object sender, RegisterSubCategoriesEvent e)
 #endif
         }
 
-        private void SetupTexControls(RegisterCustomControlsEvent e, MakerCategory makerCategory, BaseUnityPlugin owner, string clothesId, string title = "Overlay textures", bool addSeparator = false)
+        private void SetupTexControls(RegisterCustomControlsEvent e, MakerCategory makerCategory, BaseUnityPlugin owner, string clothesId, string title = "Overlay textures", bool addSeparator = false, string colormaskId = null)
         {
             var isMask = KoiClothesOverlayController.IsMaskKind(clothesId); // todo false in ai hs
             var texType = isMask ? "override texture" : "overlay texture";
+            texType = colormaskId != null ? "override texture" : texType;
+
+            var realClothesId = clothesId;
+            clothesId = colormaskId == null ? clothesId : colormaskId;
```

Comment:
This is a bit confusing now, I would add a `isColorMask` variable and compare to that rather than have all the null checks.